core/src/main/resources/org/apache/spark/ui/static/executorspage-template.html

```diff
@@ -24,7 +24,7 @@ <h4 style="clear: left; display: inline-block;">Summary</h4>
         <th></th>
         <th>RDD Blocks</th>
         <th><span data-toggle="tooltip"
-                  title="Memory used / total available memory for storage of data like RDD partitions cached in memory. ">Storage Memory</span>
+                  title="Memory used / total available memory for storage of data like RDD partitions cached in memory. ">Storage Memory used/total</span>
```

Comment:
I don't disagree your comments, but from my point it is slightly duplicated and not so necessary. And I believe most of the users understand what this column mean. So I'm conservative of such change.